src/app/api/getStakerPoints.ts

```diff
@@ -0,0 +1,29 @@
+import { encode } from "url-safe-base64";
+
+import { apiWrapper } from "./apiWrapper";
+
+export interface StakerPoint {
+  staker_btc_pk: string;
+  polkadots: number;
+}
+
+export const getStakerPoints = async (
+  stakerBtcPk: string | string[],
+): Promise<StakerPoint[]> => {
+  const params: Record<string, string> = {};
+
+  if (Array.isArray(stakerBtcPk)) {
+    params.staker_btc_pk = stakerBtcPk.map(encode).join(",");
+  } else {
+    params.staker_btc_pk = encode(stakerBtcPk);
+  }
+
+  const response = await apiWrapper(
+    "GET",
+    "/v1/points/stakers",
+    "Error getting staker points",
+    params,
+  );
+
+  return response.data;
+};
```

Comment:
Commenting here as I can't comment on the `src/app/assets/coins.svg` image: Let's rename the file to `src/app/assets/points.svg` to avoid any form of confusion.